src/libs/actions/Task.js

```diff
@@ -551,8 +551,9 @@ function getAssignee(details) {
  * */
 function getShareDestination(reportID, reports, personalDetails) {
     const report = lodashGet(reports, `report_${reportID}`, {});
+    const personalDetail = personalDetails[0] || {};
     return {
-        icons: ReportUtils.getIcons(report, personalDetails),
+        icons: ReportUtils.getIcons(report, personalDetails, UserUtils.getAvatar(personalDetail.avatar, personalDetail.accountID), true),
```

Comment:
Here `getAvatar` will always return the fallback svg because of the **undefined** value of `personalDetail`.

Passing `isPayer` is always to **true** is off for me. We only need the `isPayer` to **true** when the report is an IOU. So it's better to pass the value from `isIOUReport`.